oak/server/asylo/asylo_oak_docker.bzl

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2020 The Project Oak Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Rule for generating an Oak application Docker image."""
+
+load("@io_bazel_rules_docker//container:container.bzl", "container_image")
+
+def asylo_oak_docker(name, application, ports):
+    """Generates an Oak application Docker image.
```

Comment:
I guess this really means that there is an asylo loader that loads the application. Perhaps mention something about that.

Reply:
Added